src/miner.cpp

```diff
@@ -143,6 +143,7 @@ std::unique_ptr<CBlockTemplate> BlockAssembler::CreateNewBlock(const CScript& sc
     // Add dummy coinbase tx as first transaction
     pblock->vtx.emplace_back();
     pblock->fProofOfStake = fProofOfStake;
+	if (fProofOfStake) { pblock->vtx.emplace_back(); } // Add placeholder tx - will be replaced with CoinStake tx.
```

Comment:
You also need to add a dummy entry to vTxFees and vTxSigOpsCost

Reply:
@codeofalltrades was this part of the code meant to be replaced? Now it only add to fees and sigopscost. I gave a positive code review because I was not sure the intention, without testing this myself, it seems like it may no longer be doing what was intended?